### ISSUE TITLE
Values of m for SphereFace loss

### ISSUE THREAD

**Issue by RitiP** [ydwen/opensphere#11]
Hi,

Thank you for sharing the code for SphereFace in Pytorch! Very interesting work!
I have been using your code for SphereFace loss and was wondering how do you arrive at the value of m. I want to explore the usage of this loss function for different datasets(AWA, SUN etc.) and have currently used only the default values(s=30, m=1.5). 
It'd be great if you can elaborate how one can arrive at the value of m and tune it for different datasets.
Thanks again!
